### PR TITLE
Added support for Protonet

### PR DESCRIPTION
**Pull request by christianseel**
This pull request adds support for Protonet.

seda_digital_gmbh___co__kg___protonet_ _und_repositories_und_1__bash

Integration works with "Protonet Live" (Hosted Service: https://protonet.com/de/testen/) and with your personal local Protonet Server. Custom Domains can be added via the Toggl button settings.

### CODE REVIEW

**Comment by dooart** [comment]
Hi, Christian! Thank you for the PR. We will review it as soon as we can. 👍

I already took a quick look at the code and found two lines with a `console.log`. Do you really need those?

**Comment by christianseel** [comment]
@dooart Oops! Of course not! Removing commit coming in a moment…

**Comment by christianseel** [comment]
Updated the PR with a feature request from our team: The task in Protonet is now automatically set to "in progress" when the toggl button is clicked.

File: src/scripts/content/protonet.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    status.dispatchEvent(event);
+  });
+});
+
+// toggl button for group main naviagtion
+togglbutton.render('#project-header-intro:not(.toggl)', {observe: true}, function (elem) {
+  var link,
+    project = $('.project-title .title', elem).textContent.trim(),
+    href = $('.project-navigation li:first-child a', elem).getAttribute('href'),
+    id = href.split("/").pop(-1),
```
Comment (LulzAugusto) [inline src/scripts/content/protonet.js]: ☝️

**Comment by bryanph** [comment]
Hey @christianseel, we decided to clear up old PRs to make way for some big refactoring that is upcoming. If you wish to continue work on this PR please feel free to reopen or add an issue for adding Protonet support!

**Pull request closed without merging**